src/index.spec.js

```diff
@@ -336,6 +336,7 @@ describe('Dropzone', () => {
       );
       dropzone.find(ChildComponent).simulate('dragEnter', { dataTransfer: { files } });
       dropzone.simulate('dragEnter', { dataTransfer: { files } });
+      dropzone.simulate('dragEnter', { dataTransfer: { files } }); // make sure we de-dupe events
```

Comment:
We want to make sure that even if the browser sends us duplicate `dragEnter` events for the same element, which it might, we'll still handle it correctly without the `dragTargets` list growing too large. It would also be possible to make the `dragEnter` just be a simple push and rely on the filter in `dragLeave` to remove any duplicates later, up to you.